client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/IndicesClientDocumentationIT.java

```diff
@@ -749,17 +749,31 @@ public void testUpdateAliases() throws Exception {
         {
             // tag::update-aliases-request
             IndicesAliasesRequest request = new IndicesAliasesRequest(); // <1>
-            AliasActions aliasAction = new AliasActions(AliasActions.Type.ADD).index("index1").alias("alias1"); // <2>
+            AliasActions aliasAction =
+                    new AliasActions(AliasActions.Type.ADD)
+                    .index("index1")
+                    .alias("alias1"); // <2>
```

Comment:
+1, I'm in favour of changing the indentation.
Unfortunately we don't seem to have a way to catch accidental reformatting of these sections in the future. Just a remark, nothing to change here I think.

Reply:
Unfortunately, we don't. The only way to do that is to use the `//@formatter:off` for the snippet sections but the tags are not activated by default ( at least in eclipse, but I would guess in IntelliJ is the same). I would say one more reason to consider using a [simple formatter for ES](https://github.com/elastic/elasticsearch/issues/28519) ;)